ext/zend_test/tests/gh8575.phpt

```diff
@@ -0,0 +1,13 @@
+--TEST--
+CLI: stderr is available in mshutdown
+--SKIPIF--
+<?php if (php_sapi_name() != "cli")) die('skip: cli test only'); ?>
+--EXTENSIONS--
+zend-test
```

Comment:
The ext is `zend_test` with underscore - this should just skip it.

Reply:
Right, but only as of PHP 8.1.0 (what this PR is about). Previously, the extension name was indeed [zend-test](https://github.com/php/php-src/blob/84ea0aa6840b814f84b241e151472a352616b314/ext/zend_test/test.c#L703).

Reply:
🤦🏻 and thanks!